static/js/browser/statvar_hierarchy.tsx

```diff
@@ -180,14 +187,14 @@ export class StatVarHierarchy extends React.Component<
           rootSVGs,
         });
         const pathPromises: Promise<string[]>[] = [];
-        if (this.props.svs) {
-          for (const sv of this.props.svs) {
+        if (this.props.selectedSVs) {
```

Comment:
since the promises in this if statement don't depend on the root stat var groups data, is there a reason for nesting these promises and only running them once the root stat var groups data has been retrieved?

Reply:
Good point. Now fetch them at the same time.